### DISCUSSION TITLE
Why use `z/Z` as default keys in operator-pending mode?

### DISCUSSION THREAD

**Discussion by molleweide** [ggandor/lightspeed.nvim#80] [General]
Hi again,

I would like to ask why you decided to have operator pending mode mapped to `z/Z`?

**Comment by ggandor** [comment]
Hi! The idea is that in operator-pending mode `s` and `S` might be more useful for a "surround" plugin (like vim-surround or vim-sandwich). Sneak does the same thing with its default keymaps by the way. Moreover, we have even more reason to do so than Sneak, as we have `x/X` motions too, and I think they make very nice pairs with `z/Z`. Just as "x" is to the right of "z" (on QWERTY at least), the motion `x/X` pushes the end/start of the operated area forward (to the right).

**Reply by molleweide** [reply]
ahaa so then I have a follow up question:

I am using vim-surround by Tim Pope and I had it mapped to Z in all modes in favor of Sneak. Would you recommend me to unmap
surround to its default. Will this be fully compatible with lightspeed?

**Reply by ggandor** [reply]
Then I guess you can just remap Lightspeed to `s/S` in operator-pending mode too, whatever you prefer.
Note that vim-surround will conflict with lightspeed `S` in visual mode, if you keep the defaults. Related: https://github.com/ggandor/lightspeed.nvim/discussions/83
